scripts/core/enqueue.yml

```
@@ -54,6 +54,7 @@ returns:
     - $X_RESULT_ERROR_INVALID_QUEUE
     - $X_RESULT_ERROR_INVALID_KERNEL
     - $X_RESULT_ERROR_INVALID_EVENT
+    - $X_RESULT_ERROR_INVALID_EVENT_WAIT_LIST
```

```suggestion
    - $X_RESULT_ERROR_INVALID_EVENT_WAIT_LIST
        - "phEventWaitList == NULL && numEventsInWaitList > 0"
        - "phEventWaitList != NULL && numEventsInWaitList == 0"
```

I saw @kbenzie do something like this [here](https://github.com/oneapi-src/unified-runtime/pull/275/files#diff-49df2b979e9b956e061467dd2f8e1a6a07d5a2ac09f33acf90dac1603cb3ceefR141). I think it would be nice adding this so that the conditions for this error code are documented per entry-point. Not 100% sure though.

---

I think that makes sense. I added the details to every entrypoint

---

Seems like this is used as input for validation layers :O
Need to update it again and keep only the code conditions

---

Seems to be passing CI now :+1:

---

nice!

---

nice!